tox.ini

```diff
@@ -82,15 +82,32 @@ deps =
 
     mock
     lxml
+    coverage
+
+extras = all
```

Comment:
these changes look spurious: the py27-old env deliberately uses its own pip incantations rather than letting tox do them.

Reply:
Nyargh, I swear it didn't work without these changes so I added them to try and get them into line but I've removed them and now it works and I'm very sad and confused.